modules/ROOT/pages/migration/index.adoc

```diff
@@ -18,533 +18,56 @@ npm update @neo4j/graphql
-
-
-=== Removed `connectComponentsPlanner` from `cypherQueryOptions`
-
-Following the deprecation of the link:https://neo4j.com/docs/cypher-manual/current/planning-and-tuning/query-tuning/#cypher-connect-components-planner[`connectComponentsPlanner`], the field is no longer accepted in the GraphQL context and is removed from the `cypherQueryOptions` object.
-
-== Additions
-
-=== Added the `_EQ` filter as an alternative to the deprecated implicit equal filters
-
-The `count_EQ` filter is now available as an alternative to the deprecated `count` filter.
-
-[cols="1,1"]
-|===
-|Before | Now
-
-a|
-[source, graphql, indent=0]
-----
-{
-  movies(where: { actorsAggregate: { count: 10 } }) {
-    title
-  }
-}
-----
-a|
-[source, graphql, indent=0]
-----
-{
-  movies(where: { actorsAggregate: { count_EQ: 10 } }) {
-    title
-  }
-}
-----
-|===
-
-=== Added the `_EQ` filter as an alternative to the deprecated implicit "equal" filter
-
-The `_EQ` filter is now available as an alternative to the deprecated implicit "equal" filter.
-
-[cols="1,1"]
-|===
-|Before | Now
-
-a|
-[source, graphql, indent=0]
-----
-{
-  movies(where: { title: "The Matrix" }) {
-    title
-  }
-}
-
-----
-a|
-[source, graphql, indent=0]
-----
-{
-  movies(where: { title_EQ: "The Matrix" }) {
-    title
-  }
-}
-----
-|===
-
-== Deprecations and warnings
-
-=== Implicit equality filters are deprecated
-
-The following implicit equality filters are deprecated: 
-
-  - `{ name: "Keanu" }`
-  - `{ count: 10 }` 
-
-You can achieve the same by using `{ name_EQ: "Keanu" }` and `{ count_EQ: 10 }`.
-The deprecated quality filters will be removed in version 7.x.
-
-[cols="1,1"]
-|===
-|Before | Now
-
-a|
-[source, graphql, indent=0]
-----
-query {
-  users(where: { name: "John" }) {
-    id
-    name
-  }
-}
-----
-a|
-[source, graphql, indent=0]
-----
-query {
-  users(where: { name_EQ: "John" }) {
-    id
-    name
-  }
-}
-----
-|===
-
-=== `@node` will have to be explicitly defined
-
-In the future, types without the `@node` directive will no longer be treated as Neo4j nodes.
-In version 6.0.0, it's not required to specify every GraphQL type representing a Neo4j node with the `@node` directive, however it's recommended to do so and a warning is raised if you don't.
-There are genuine cases in which a `@node` should not be used, e.g., for types returned by @cypher fields which are not Neo4j nodes.
-
-[cols="1,1"]
-|===
-|Before | Now
-
-a|
 [source, graphql, indent=0]
 ----
 type Movie {
-    title: String
-    actors: [Person!]! @relationship(type: "ACTED_IN", direction: IN)
-}
-
-type Person {
-    name: String
+    title: String! @unique()
 }
 ----
-a|
-[source, graphql, indent=0]
-----
-type Movie @node {
-    title: String
-    actors: [Person!]! @relationship(type: "ACTED_IN", direction: IN)
-}
 
-type Person @node {
-    name: String
-}
-----
-|===
 
-=== Deprecated the implicit equality filters
+=== Non-list relationships
```

Comment:
we have "non-list relationships", "single element relationships" and "1-to-1 relationships" in this section, referring to the same(?)

we should pick one ("1-to-1 relationships?)

Reply:
I think the more accurate is non-list relationships. As 1-1 refers only to a subset of the cases.

Yeah, my bad, I'll change them all to be the same